packages/expo-barcode-scanner-interface/android/src/main/java/expo/interfaces/barcodescanner/BarCodeScannerResult.java

```diff
@@ -1,12 +1,23 @@
 package expo.interfaces.barcodescanner;
 
+import android.graphics.Point;
+
+import java.util.List;
+
 public class BarCodeScannerResult {
+  private int mWidth;
+  private int mHeight;
   private int mType;
   private String mValue;
+  private List<Integer> mCornerPoints;
 
-  public BarCodeScannerResult(int type, String value) {
+
+  public BarCodeScannerResult(int type, String value, List<Integer> cornerPoints, int height, int width) {
```

Comment:
Let's remember:
- `int mType`
- `String mValue`
- origin, size, corner points (or just corner points + export methods returning origin and size based on corner points), note that such way points will have to be already converted to proper scale